app/app.go

```diff
@@ -97,9 +97,12 @@ func (a *App) Start(ctx context.Context) error {
 	if a.cfg.GetBool(config.Ipfsnode, true) {
 		// setup local ipfs node
 		node := node.NewIpsNode(a.cfg)
-
-		err = a.RunAsync("IpfsNode", node, func() error {
-			return node.Start(ctx)
+		a.RunAsync("IpfsNode", node, func() error {
+			if err := node.Start(ctx); err != nil {
+				log.Error("Error starting embedded IPFS node", err)
+				return err
+			}
+			return nil
 		})
 		if err != nil {
 			return err
```

Comment:
@dmerrill6 

What do you think about adding `log.Error("Error starting embedded IPFS node", err)` here?

It should be the same but does not break the `err = a.RunAsync()` pattern.

Reply:
This was a leftover from the merge with master. Fixed in 6a1fced.